reporthandling/results/v1/reportsummary/datastructures.go

```diff
@@ -77,3 +78,35 @@ type PostureCounters struct {
 	SkippedCounter  int `json:"skipped"`
 	ExcludedCounter int `json:"excluded"` // Deprecated
 }
+
+type VulnerabilitySummary struct {
+	MapsSeverityToSummary MapsSeverityToSummary `json:"MapsSeverityToSummary"`
+	CVESummary            []CVESummary          `json:"CVEs"`
+	PackageScores         PackageScores         `json:"PackageScores"`
+	Images                []string              `json:"Images"`
+}
+
+type MapsSeverityToSummary map[string]*SeveritySummary
+
+type SeveritySummary struct {
+	NumberOfCVEs        int `json:"NumberOfCVEs"`
```

Comment:
Please change all json fields to lowercase.
e.g.
`json:"NumberOfCVEs"` -> `json:"numberOfCVEs"`

This can remain the same:
`json:"CVEs"`

Reply:
Done!